schema/sql/api.sql

```diff
@@ -31,9 +31,23 @@ grant usage on sequence "user_id" to "api-user-create";
 grant "api-user-create" to "api";
 grant "api-anon" to "api-user-create";
 
+create role "api-user-manage";
+comment on role "api-user-manage" is
+    $$Intended for use with user management systems$$;
+grant usage on sequence "user_id" to "api-user-manage";
+grant select,insert,update on table
+    public."passwd",
+    public."ssh_public_key",
+    public."openpgp_public_key"
+to "api-user-manage";
+grant "api-user-manage" to "api";
+grant "api-anon" to "api-user-manage";
+grant "api-user-create" to "api-user-manage";
+
 create schema v1;
 grant create,usage on schema v1 to api;
 grant usage on schema v1 to "api-anon";
+grant create,usage on schema v1 to "api-user-manage";
```

Comment:
Why are you granting `create` here?

Reply:
Because this application creates new users.

Reply:
This is create on the schema: it's the ability to create new tables.

Reply:
good catch. Fixed.